docs/azure.rst

```diff
@@ -7,7 +7,7 @@ Azure Cloud
 Requirements
 ============
 
-The support for Azure Cloud requires Nextflow version ``21.02.0-edge`` or later. If you don't have it installed
+The support for Azure Cloud requires Nextflow version ``21.04.1-edge`` or later. If you don't have it installed
 use the following command to download it in your computer::
```

Comment:
This was changed in the current master, better to revert the change to avoid conflicts

Reply:
done